mopidy/http/actor.py

```diff
@@ -177,10 +177,23 @@ def _get_static_request_handlers(self):
         return result
 
     def _get_mopidy_request_handlers(self):
+        sites = [app["name"] for app in self.apps + self.statics]
+
+        default_webclient = self.config["http"]["default_webclient"]
+        if default_webclient not in sites:
+            logger.warning(
+                "Invalid default_webclient %s, "
+                "Ignoring unknown default webclient",
+                default_webclient,
+            )
+
+            default_webclient = "mopidy"
+        logger.debug("Default webclient is %s", default_webclient)
+
         return [
             (
-                "/",
+                r"/",
                 tornado.web.RedirectHandler,
-                {"url": "/mopidy/", "permanent": False},
+                {"url": "/{}/".format(default_webclient), "permanent": False},
```

Comment:
We are trying to make use of Python 3's f-strings where this would be written as `f"/{default_webclient}/"`

Reply:
I'm also surprised black didn't change all this to single quotes.

Reply:
done !